src/components/TextListItem.tsx

```diff
@@ -107,15 +107,14 @@ export const TextListItem: NamedExoticComponent<Props> & {
       Component={Touchable}
       accessibilityLabel={`(${title}) ${consts.a11yLabel.button}`}
     >
-      {leftIcon ||
-        (leftImage && !!picture?.url ? (
-          <Image
-            source={{ uri: picture.url }}
-            style={[styles.smallImage, withCard && styles.withBigCardStyle]}
-            borderRadius={withCard ? normalize(8) : undefined}
-            containerStyle={styles.smallImageContainer}
-          />
-        ) : undefined)}
+      {(leftIcon || leftImage) && !!picture?.url ? (
```

Comment:
the condition changed accidentally? if a `leftIcon` is give, it should be rendered. everything else was the "else" part, which is now changed?

Reply:
I'm sorry. I misunderstood the logic here. I'll undo the changes.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/4550329f16ccdef1de562c18775b5f594b9564b4